### ISSUE TITLE
Add a rule to forbid types

### ISSUE THREAD

**Issue by j-f1** [gajus/eslint-plugin-jsdoc#134]
TypeScript uses JSDoc for documentation purposes, but since it already has a type system, the type declarations are not useful and can be inferred from the source code. Could there be a rule, perhaps `jsdoc/no-types`, that prohibits any specification of types in the comment?

```ts
// Valid
/**
 * Adds two numbers
 * @param a The first number
 * @param b The second number
 * @returns The sum
 */
function add(a: number, b: number) {
  return a + b
}

// Invalid
/**
 * Adds two numbers
 * @param {number} a The first number
error     ~~~~~~~~
 * @param {number} b The second number
error     ~~~~~~~~
 * @returns {number} The sum
error       ~~~~~~~~
 */
function add(a: number, b: number) {
  return a + b
}

```
